spec/user_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe User do
+  let!(:user) { User.create( name: 'New User',
+                      username: 'whoami',
+                      email: 'user@email.com',
+                      password: 'password',
+                      password_confirmation: 'password')
+              }
```

Comment:
Lint/BlockAlignment: `end` at 7, 14 is not aligned with `let!(:user) { User.create( name: 'New User',` at 2, 2.